maven-release-manager/src/main/java/org/apache/maven/shared/release/phase/AbstractRewritePomsPhase.java

```diff
@@ -530,25 +537,25 @@ else if ( mappedVersion.equals( propertyValue ) )
                             {
                                 // this property may have been updated during processing a sibling.
                                 logInfo( result, "  Ignoring artifact version update for expression " + rawVersion
-                                    + " because it is already updated" );
+                                        + " because it is already updated" );
                             }
                             else if ( !mappedVersion.equals( rawVersion ) )
                             {
                                 if ( mappedVersion.matches( "\\$\\{project.+\\}" )
-                                    || mappedVersion.matches( "\\$\\{pom.+\\}" )
-                                    || "${version}".equals( mappedVersion ) )
+                                        || mappedVersion.matches( "\\$\\{pom.+\\}" )
+                                        || "${version}".equals( mappedVersion ) )
```

Comment:
Can you add a comment here that `pom.` and prefixless is gone in Maven 4?

Reply:
Added

Reply:
BTW, this is yet another indicator release plugin does too much: it seems to duplicate too much logic that is in core...